youtube_dl/extractor/funimation.py

```diff
@@ -110,11 +110,24 @@ def _search_kane(name):
             headers = {}
             if self._TOKEN:
                 headers['Authorization'] = 'Token %s' % self._TOKEN
-            sources = self._download_json(
-                'https://www.funimation.com/api/showexperience/%s/' % video_id,
-                video_id, headers=headers, query={
-                    'pinst_id': ''.join([random.choice(string.digits + string.ascii_letters) for _ in range(8)]),
-                })['items']
+            try:
+                sources = self._download_json(
+                    'https://www.funimation.com/api/showexperience/%s/' % video_id,
+                    video_id, headers=headers, query={
+                        'pinst_id': ''.join([random.choice(string.digits + string.ascii_letters) for _ in range(8)]),
+                    })
+                sources = sources['items']
+            except KeyError:
+                if 'errors' in sources:
```

Comment:
If the key `items` doesn't exist, we except this and try to figure out what the error is. If there is at least one error, try to get the `detail` and display it to the user. If there's an error without a `detail`, just spit out the whole error to the user. Could perhaps fall back even further if there's not an `errors[0]` at all and simply spit out the whole `sources` but that's probably a bit overkill? If there's not an `errors[0]`, odds are the user isn't going to have any idea how to fix it.